apigee_sync_datascope_cache.go

```diff
@@ -0,0 +1,93 @@
+package apidApigeeSync
```

Comment:
Also, there should be a datascope_cache_test.go to verify just the cache operations.